presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/TestPostgreSqlIntegrationSmokeTest.java

```diff
@@ -278,6 +278,23 @@ public void testInsertIntoNotNullColumn()
         assertUpdate("DROP TABLE test_insert_not_null");
     }
 
+    @Test
+    public void testColumnComment()
+            throws Exception
+    {
+        execute("CREATE TABLE tpch.test_column_comment (col1 bigint, col2 bigint, col3 bigint)");
+        execute("COMMENT ON COLUMN tpch.test_column_comment.col1 IS 'test comment'");
+        execute("COMMENT ON COLUMN tpch.test_column_comment.col2 IS ''");
```

Comment:
```suggestion
        execute("COMMENT ON COLUMN tpch.test_column_comment.col2 IS ''"); // it will be NULL, PostgreSQL doesn't store empty comment
```